transcoder/main.py

```diff
@@ -105,6 +105,9 @@ def main():
                                            'source data. Default behavior is to create resources for each message '
                                            'type before messages are processed. Particularly useful when working with '
                                            'FIX but only processing a limited set of message types in the source data')
+    output_options_group.add_argument('--frame_only', action='store_true',
+                                      help='Flag indicating that transcoder should only frame messages into a data '
```

Comment:
"Frame messages to an output source" might describe the intention more accurately